src/Helper/DbStringHelper.php

```diff
@@ -48,6 +49,20 @@ public static function baseName(string $path): string
         return $path;
     }
 
+    /**
+     * Returns a value indicating whether an SQL statement is for read purpose.
+     *
+     * @param string $sql The SQL statement.
+     *
+     * @return bool Whether an SQL statement is for read purpose.
+     */
+    public static function isReadQuery(string $sql): bool
```

Comment:
Incorrect class for this method. Please add new Helper. DbQueryHelper as example